pkg/api/types.go

```diff
@@ -1277,7 +1277,7 @@ type BoundPods struct {
 	Host string `json:"host"`
 
 	// Items is the list of all pods bound to a given host.
-	Items []BoundPod `json:"items"`
+	Items []Pod `json:"items"`
```

Comment:
This is something that is missing in this PR :)
Basically, in this proof of concept I'm still using "BoundPods" which I should change to "Pods".
Will change this on Monday.